php_sql/task1/index.php

```diff
@@ -0,0 +1,52 @@
+<?php 
+    Include "./dbOperation.php";
+    $valid=array();
+    $errors=array();
+    $maxSizeCourse =128;
+    $maxSizeName=128;
+    $maxLengthDescription=1024;
+    function validate ($name, $limit, &$valid, &$errors)
+    {
+        $check=$_POST["$name"];
+        if (!$check)
+        {
+            $errors["$name"]="$name е задължително поле";
+        }
+        elseif(strlen("$name")>$limit)
+        {
+            $errors["$name"]="$name трябва да бъде не повече от $limit символа";
+        }
+        else
+        {
+            $valid["$name"]=$check;
+        }
+    }
+    function check ($name, &$valid, &$errors)
+    {
+        $var=$_POST["$name"];
+        if (!$var)
+        {
+            $errors["$name"]='Полето е задължително';
+        }
+        else
+        {
+            $valid["$name"]=$var;
+        }
+    }
+    if ($_POST) 
+    {
+        validate("title", $maxSizeCourse, $valid, $errors);
+        validate("name", $maxSizeName, $valid, $errors);
+        validate("description", $maxLengthDescription, $valid, $errors);
+    }
+        if(isset($_POST['submit']) && count($errors)==0)
+        {
+          query();
+          echo "Success!";
+         }
+         else 
+         {
+             echo "There is invalid input! Please enter the information again! You will be redirected in 2 seconds!";
+             header( "refresh:2; url=index.html" );
```

Comment:
It would be better if you just output a url so the user can navigate by himself. Also 2 seconds it too short, the user may not have time to see the message.